app/static/tests/unit/components/run/runPlot.test.ts

```diff
@@ -276,7 +275,7 @@ describe("RunPlot", () => {
                 fadePlot: false,
                 graphIndex: 0,
                 graphConfig
-            },
+            } as any,
```

Comment:
Why do you need these `as any`s - is this not providing all the props the RunPlot expects? I suppose could spread an empty props object here if so, but probably not worth it!

Reply:
`linkedXAxis` was the issue in this one! yh just thought it wasnt worth it to add it cos we dont use `linkedXAxis` in most of these tests